src/lib/OpenEXRCore/parse_header.c

```diff
@@ -2364,6 +2364,7 @@ internal_exr_compute_chunk_offset_size (struct _internal_exr_part* curpart)
             case EXR_COMPRESSION_B44A:
             case EXR_COMPRESSION_DWAA: linePerChunk = 32; break;
             case EXR_COMPRESSION_DWAB: linePerChunk = 256; break;
+            case EXR_COMPRESSION_ZSTD: linePerChunk = 256; break;
```

Comment:
shouldn't that be 32 instead of 256 ?

Reply:
A debug change: Makes the crash happen faster. I will revert it once we fix it.